src/components/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { Route, Routes } from "react-router";
+import Articles from "./Articles";
+import SingleArticle from "./SingleArticle";
+import Header from "./Header";
+import ScrollToTop from "./ScrollToTop";
+
+function App() {
+  return (
+    <main className="bg-neutral-100">
+      <Header />
+      <ScrollToTop />
+      <Routes>
+        <Route path="/" element={<Articles />} />
+        <Route path="/:article_id" element={<SingleArticle />} />
```

Comment:
I don't think I like the URL without `/articles` at the beginning... the reason being it's not RESTful, semantic or scalable.